mirth_exporter.go

```diff
@@ -182,21 +241,22 @@ func (e *Exporter) readChannelStats(lines []string, ch chan<- prometheus.Metric)
 
 func main() {
 	var (
-		listenAddress = flag.String("web.listen-address", ":9140",
+		listenAddress = flag.String("web.listen-address", ":9041",
 			"Address to listen on for telemetry")
-		metricsPath = flag.String("web.telemetry-path", "/metrics",
+		metricsPath = flag.String("web.telemetry-path", "/internal/metrics",
 			"Path under which to expose metrics")
-		mccliConfigPath = flag.String("mccli.config-path", "./mirth-cli-config.properties",
-			"Path to properties file for Mirth Connect CLI")
-		mccliJarPath = flag.String("mccli.jar-path", "./mirth-cli-launcher.jar",
-			"Path to jar file for Mirth Connect CLI")
-	)
+    mccliJarPath = flag.String("mccli.jar-path", "./../opt/connect/mirth-cli-launcher.jar",
+        "Path to jar file for Mirth Connect CLI")
+  )
 	flag.Parse()
 
-	exporter := NewExporter(*mccliJarPath, *mccliConfigPath)
+	promlogConfig := &promlog.Config{}
```

Comment:
all of the promlog and log changes are just to update to prometheus' recent logging libraries.